__test__/shop.test.js

```diff
@@ -0,0 +1,89 @@
+var Shop = require('../src/shop');
+var Item = require('../src/item');
+
+describe("Gilded Rose", function() {
+  it('Name does not change after update', () => {
+    const gildedRose = new Shop([ new Item("foo", 0, 0) ]);
+    const items = gildedRose.updateQuality();
+    expect(items[0].name).toBe("foo");
```

Comment:
Just a suggestion: It may be good to separate out 1. The Test setup (line 6 in this case), 2. The thing that executes whatever you're testing (7), and 3. What you're testing (8) by adding spaces in between.

If there's more than 1 line for each thing, we usually group them together and put a line in between each thing to separate them into the 3 things above.